etna/bin/etna

```diff
@@ -1,12 +1,26 @@
 #!/usr/bin/env ruby
 
-require 'bundler'
-Bundler.require :default
+if ENV['ETNA_GEM_DEVELOPMENT']
+  require 'bundler'
+  Bundler.require :default
+end
 
 require 'yaml'
 require_relative '../lib/etna'
 require_relative '../lib/commands'
 
-config = YAML.load(File.read(File.expand_path("../../config.yml",__FILE__)))
+config = if File.exists?(EtnaApp.config_file_path)
+  YAML.load(File.read(EtnaApp.config_file_path))
+else
+  { production: {} }
```

Comment:
Can we add also a default `development` key to the config hash? If someone makes a mistake (like me) on the first time using the config command, the config command breaks and I have to manually remove `etna.yml`:

```
○ → etna config set htps://polyphemus.ucsf.edu
Traceback (most recent call last):
	6: from /app/bin/etna:18:in `<main>'
	5: from /app/lib/etna/application.rb:102:in `run_command'
	4: from /app/lib/commands.rb:55:in `execute'
	3: from /app/lib/etna/clients/polyphemus/workflows/set_configuration_workflow.rb:31:in `update_configuration_file'
	2: from /app/lib/etna/clients/polyphemus/client.rb:23:in `configuration'
	1: from /app/lib/etna/client.rb:38:in `get'
/app/lib/etna/client.rb:107:in `query_request': undefined method `request_uri' for #<URI::Generic htps://polyphemus.ucsf.edu/configuration?> (NoMethodError)

 2020-10-23 11:42:18 ⌚  b63aec446a91 in /
○ → etna config set https://polyphemus.ucsf.edu
Traceback (most recent call last):
	5: from /app/bin/etna:18:in `<main>'
	4: from /app/lib/etna/application.rb:96:in `run_command'
	3: from /app/lib/commands.rb:60:in `setup'
	2: from /app/lib/etna/command.rb:258:in `setup'
	1: from /app/lib/etna/application.rb:49:in `configure'
/app/lib/etna/application.rb:76:in `config': undefined method `[]' for false:FalseClass (NoMethodError)

 2020-10-23 11:42:25 ⌚  b63aec446a91 in /
○ → rm ~/etna.yml 
```

I think it's because the default environment EtnaApplication looks for is `development`, which doesn't exist in `etna.yml`.

Reply:
Right.  Two things -- one, it probably shouldn't create a file if the request fails (it touches an empty file), and two, if a file is poorly formatted I can probably guard against that better.  I'll have a fix soon.

Reply:
https://github.com/mountetna/monoetna/pull/118/commits/d7276500aef47e9f57c85cbe7b81020ad33ff7e8#diff-d0be4b3b7a734970e675b48dc7ae203413a1b112a341d1887c9af0423b0368b2R26